client/src/pages/practice.js

```diff
@@ -0,0 +1,76 @@
+<form>
```

Comment:
Bones of the student form, creating text fields for input.

Reply:
Is this component a duplicate of the student component, or am I missing something?

Reply:
I deleted this. Should not have made it into this PR